universe/ValueRefs.cpp

```diff
@@ -316,14 +318,14 @@ MeterType NameToMeter(const std::string_view name) {
     for (std::size_t i = 0; i < NAME_BY_METER.size(); i++)
     {
         if (NAME_BY_METER[i] == name)
-            return static_cast<MeterType>(i);
+            return static_cast<MeterType>(i - 1);
     }
 
     return MeterType::INVALID_METER_TYPE;
 }
 
 const std::string& MeterToName(const MeterType meter) {
-    return (meter == MeterType::INVALID_METER_TYPE) ? EMPTY_STRING : NAME_BY_METER[static_cast<std::size_t>(meter)];
+    return NAME_BY_METER[static_cast<int>(meter) + 1];
```

Comment:
probably better cast to `std::underlying_type_t<MeterType>`

Reply:
Yup, changed.